### ISSUE TITLE
Accessibility: Missing visual focus on Storefinder page

### ISSUE THREAD

**Issue by KateChuen** [SAP/cloud-commerce-spartacus-storefront#6589]
**Environment Details**
Spartacus: epic/accessibility branch
Browser: Chrome & Safari & Firefox
Device: mac
Happens on: http://dev-com-3.accdemo.b2c.ydev.hybris.com:4200/electronics-spa/en/USD

**Steps to Reproduce**
- Go to StoreFinder page. 
- Type in a location -> Missing visual focus on the "Search" icon.
<img width="1235" alt="Screen Shot 2020-02-17 at 9 17 01 AM" src="https://user-images.githubusercontent.com/45409515/74661785-a1b92180-5166-11ea-8561-04f4a15fa38c.png">

- Press ENTER
- Select one store -> Missing visual focus on stores.

<img width="1200" alt="Screen Shot 2020-02-17 at 9 16 46 AM" src="https://user-images.githubusercontent.com/45409515/74661740-8c43f780-5166-11ea-818d-5becf74e6d4a.png">

- Click on "View All Stores"
- Tab to the result "Japan (49)"
- On SAFARI only, missing visual focus
<img width="1321" alt="Screen Shot 2020-02-17 at 9 22 07 AM" src="https://user-images.githubusercontent.com/45409515/74662048-2ad05880-5167-11ea-880a-99b9346303e4.png">

**Comment by developpeurweb** [comment]
**1. Search icon**'s focus indicator is now visible as per screenshot below, although nothing happens when `Enter` key is used, but this is unrelated to indicator's visibility, therefore new ticket created: GH-6766.

![Screen Shot 2020-02-28 at 9 08 21 PM](https://user-images.githubusercontent.com/15113219/75787461-c3a0cf80-5d34-11ea-99c1-633e928ec31d.png)

**2. Store names** are not expected to have focus, the focusable element is in fact the button (as per screenshot below), this is OK for this Accessibility milestone aimed as sighted users... but this will have to be fixed in the next milestone when we improve for Screen Readers, as right now vocalization is meaningless, a ticket was created for this specific issue: GH-6767.

![Screen Shot 2020-03-02 at 4 10 41 PM](https://user-images.githubusercontent.com/15113219/75788431-3bbbc500-5d36-11ea-9147-6acc46d3880a.png)

**3. Country link** focus indicator bug for Safari is now fixed, as per screenshot below.

![Screen Shot 2020-02-28 at 8 49 37 PM](https://user-images.githubusercontent.com/15113219/75788585-702f8100-5d36-11ea-9ee1-7ca099314968.png)

**Comment by KateChuen** [comment]
Tested on dev10. 
Verified on chrome, firefox, safari.
Verified that focus is fixed on the search icon, the "country results" on storefinder page.
